### PR TITLE
Add Support for Magicite Esper

### PR DESCRIPTION
**Pull request by dwoo415**

### CODE REVIEW

**Comment by jdel** [comment]
Sweet ! 👍

File: config/default.magicite-normal-stats.example.yml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+magicite:
+    summon: 
+      skillSpammer: false
+      duration: 0
+      gaugeFillTime: 0
```
Comment (ThauEx) [inline config/default.magicite-normal-stats.example.yml]: Please add a newline at the end of the file

File: lib/filter/magicite.js [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      mid: '',
+      'mid-mid': '',
+      right: '',
+      'right-mid': '',
+      middle: ' ',
+    }
+  });
+  
+  for (var beast in json) {
+  	var tmpRow = [];
```
Comment (ThauEx) [inline lib/filter/magicite.js]: Please the tab with spaces

File: lib/filter/magicite.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    tmpRow.push(params.atk);
+    tmpRow.push(params.def);
+    tmpRow.push(params.matk);
+    tmpRow.push(params.mdef);
+    tmpRow.push(params.mnd);
+    tmpRow.push(params.acc);
+    tmpRow.push(params.eva);
+    tmpRow.push(params.spd);
+ 
+	/**
```
Comment (ThauEx) [inline lib/filter/magicite.js]: Please the tab with spaces

File: lib/filter/magicite.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+     * Skill spammer 
+     * Allows all super skills to be instant-cast
+     * Set cost to 100 ms. (Normal cost is 2.5 secs)
+     * Number of uses set to 20
+     */
+	if (config.get('magicite.summon.skillSpammer')) {
```
Comment (ThauEx) [inline lib/filter/magicite.js]: Please the tab with spaces

Comment (ThauEx) [reply lib/filter/magicite.js]: And please replace the other tab characters with spaces too

**Comment by jdel** [comment]
@ThauEx: any chance for a 10.1 release ? I will make the docker container.

**Comment by ThauEx** [comment]
Maybe today 😉

**Comment by ThauEx** [comment]
Pushed the release yesterday.

**Pull request merged**